revscores/languages/language.py

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Do we have conventions for whitespaces, linebreaks, etc?
This line 1 seems unnecessary.